src/Mod/Path/PathScripts/PathDressupLeadInOut.py

```diff
@@ -434,7 +434,7 @@ def generateLeadInOutCurve(self, obj):
         # Add last layer
         if len(queue) > 0:
             layers.append(queue)
-            queue = []
+            del queue  # was `queue = []`, but changed due to LGTM suggestion
```

Comment:
Looks like copy-pasta to me.  I suggest the line is dropped entirely to let the GC take care of it when the method returns.

Reply:
Changed per your suggestion.